src/components/Conversation/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { Card } from '@components/UI/Card';
+import { GridItemEight, GridItemFour, GridLayout } from '@components/UI/GridLayout';
+import MetaTags from '@components/utils/MetaTags';
+import { Profile } from '@generated/types';
+import isFeatureEnabled from '@lib/isFeatureEnabled';
+import { Message } from '@xmtp/xmtp-js';
+import { useRouter } from 'next/router';
+import { FC } from 'react';
+import { APP_NAME } from 'src/constants';
+import Custom404 from 'src/pages/404';
+import { useAppStore } from 'src/store/app';
+import { useXmtpStore } from 'src/store/xmtp';
+
+interface Props {
+  profile: Profile;
+}
+
+const Conversation: FC<Props> = () => {
+  const router = useRouter();
+  const address = router.query.address as string;
+  const xmtpState = useXmtpStore((state) => state);
+  const { messages, conversations } = xmtpState;
+  const currentProfile = useAppStore((state) => state.currentProfile);
+
+  const onConversationSelected = (address: string) => {
+    router.push(address ? `/messages/${address}` : '/messages/');
+  };
+
+  if (!isFeatureEnabled('messages', currentProfile?.id)) {
+    return <Custom404 />;
+  }
+
+  return (
+    <GridLayout>
+      <MetaTags title={`Conversation • ${APP_NAME}`} />
+      <GridItemFour>
+        <Card className="h-[86vh] px-2 pt-3">
+          <div className="flex justify-between">
+            <div className="font-black text-lg">Messages</div>
```

Comment:
```suggestion
            <div className="font-bold text-lg">Messages</div>
```

Reply:
Styling is just dummy, for now, this is something we will be working on as per the designs. I suggest we ignore everything related to styling for now